Automattic/wp-calypso
# Add screen-reader-text component

Add `<ScreenReaderText />` component.

There's a tendency to do funny things in JSX to include an element with the class `screen-reader-text`. This component makes the intention explicit and saves class related linter headaches.

To test:
* Visit https://calypso.live/devdocs/design/screen-reader-text?branch=add/component-screenreadertext to see the component in action.
* Verify it renders and behaves as expected.
* Check console for errors and warnings.

Follow up from p1496953957829817-slack-calypso
